src/btree/bt_page.c

```diff
@@ -160,6 +165,18 @@ __page_merge_internal_delta_with_base_image(WT_SESSION_IMPL *session, WT_REF *re
 
     /* Perform a merge sort between the base array and the delta array. */
     while (i < base_entries && j < delta_entries) {
+#ifdef HAVE_DIAGNOSTIC
```

Comment:
This entire code path will go away in the future. Seems not worthing add the verbose message now.